guides/faq.rst

```diff
@@ -259,9 +259,12 @@ And a docker compose file looks like this:
         image: esphome/esphome
         volumes:
           - ./:/config:rw
+          - /etc/localtime:/etc/localtime:ro
```

Comment:
```suggestion
          # Use local time for logging timestamps
          - /etc/localtime:/etc/localtime:ro
```

I think the explanation below can be moved into a simple comment in the docker compose YAML.